### ISSUE TITLE
Releases for bower

### ISSUE THREAD

**Issue by WaKeMaTTa** [GetmeUK/ContentTools#401]
Hi guys,

Really nice job done here.

Can you update the release for https://bower.io/ 

Thnaks.

**Comment by anthonyjb** [comment]
Hi @WaKeMaTTa,

Sorry I'm not a bower users so I might be mistaken but if I run `bower info ContentTools` at my command linke it tells me the latest available version is `1.3.1` which is the current release. So by my reading it is up to date, am I missing something?

```
bower not-cached    https://github.com/GetmeUK/ContentTools.git#*
bower resolve       https://github.com/GetmeUK/ContentTools.git#*
bower checkout      ContentTools#1.3.1
bower resolved      https://github.com/GetmeUK/ContentTools.git#1.3.1

{
  name: 'ContentTools',
  description: 'A JS library for building WYSIWYG editors for HTML content',
  main: [
    'build/content-tools.js',
    'build/content-tools.min.css',
    'build/icons.woff',
    'build/images/drop-horz.svg',
    'build/images/drop-vert-above.svg',
    'build/images/drop-vert-below.svg',
    'build/images/video.svg'
  ],
  authors: [
    {
      name: 'Anthony Blackshaw',
      email: 'ant@getme.co.uk',
      url: 'https://github.com/anthonyjb'
    }
  ],
  license: 'MIT',
  keywords: [
    'wysiwyg',
    'inline',
    'html',
    'editor'
  ],
  homepage: 'http://getcontenttools.com/api/content-tools',
  repository: {
    type: 'git',
    url: 'git@github.com:GetmeUK/ContentTools.git'
  },
  moduleType: [
    'globals'
  ],
  ignore: [
    '**/.*',
    'node_modules',
    'bower_components',
    'test',
    'tests'
  ],
  version: '1.3.1'
}

Available versions:
  - 1.3.1
  - 1.3.0
  - 1.2.11
  - 1.2.10
  - 1.2.9
  - 1.2.8
  - 1.2.7
  - 1.2.6
  - 1.2.5
  - 1.2.4
  - 1.2.3
  - 1.2.2
  - 1.2.1
  - 1.2.0
  - 1.0.1

You can request info for a specific version with 'bower info ContentTools#<version>'
```

Ant

**Comment by WaKeMaTTa** [comment]
All right. Apologies.
